shop-ui/src/main/java/com/maxshop/fragment/ProductDetailsFragment.kt

```diff
@@ -19,6 +19,15 @@ internal class ProductDetailsFragment : BaseFragment() {
 
     override fun getBottomNavVisibility(): Boolean = false
 
+    override fun addLifecyclerObserver() {
+        super.addLifecyclerObserver()
+        lifecycle.addObserver(
+            viewModel.also {
```

Comment:
Same as above. Please divide `addObserver` and `viewModel.also` logic. There is no connection between these calls.